Source/Bridge.swift

```diff
@@ -183,3 +187,23 @@ extension Bridge: ScriptMessageHandlerDelegate {
         logger.warning("Unhandled message received: \(String(describing: scriptMessage.body))")
     }
 }
+
+private extension WKWebView {
+    /// NOTE: The async version crashes the app with `Fatal error: Unexpectedly found nil while implicitly unwrapping an Optional value`
+    /// in case the function doesn't return anything.
+    /// This is a workaround. See https://forums.developer.apple.com/forums/thread/701553 for more details.
+    @discardableResult
+    func evaluateJavaScriptAsync(_ str: String) async throws -> Any? {
+        return try await withCheckedThrowingContinuation { (continuation: CheckedContinuation<Any?, Error>) in
+            DispatchQueue.main.async {
```

Comment:
Can we accomplish the same with decorating the function with `@MainActor`?

Reply:
Good point, yes!